src/main/java/com/ankist/expensetracker/controller/ExpenseRestService.java

```diff
@@ -0,0 +1,46 @@
+package com.ankist.expensetracker.controller;
+
+import com.ankist.expensetracker.model.ExpenseDTO;
+import com.ankist.expensetracker.repository.ExpenseDAO;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+public class ExpenseRestService {
+
+    @Autowired
+    private ExpenseDAO expenseServiceDAO;
```

Comment:
Using field injection is not advised. Please use constructor injection. Also rename DAO to repository.
```suggestion
    private ExpenseDAO expenseServiceDAO;
    
    public ExpenseRestController(ExpenseDAO expenseDao) {
        this.expenseServiceDAO = expenseDao;
    }    

```